public_html/wp-content/mu-plugins/blocks/assets/src/sponsors/block-content.js

```diff
@@ -150,7 +131,7 @@ class SponsorBlockContent extends Component {
 				break;
 		}
 
-		this.setState( {
+		return( {
```

Comment:
`SHOULD` - It doesn't look like there are any early `return null` calls in this function, to avoid updating state on every render. Is that intentional?

There may be other instances of this as well.

Reply:
I think this is now irrelevant with 045adb0ad1f82836cc9946be2daf27db41555b14 and other commits where I have eliminated usage of `getDerivedStateFromProps`.